src/main/java/org/example/hn/UserController.java

```diff
@@ -0,0 +1,80 @@
+package org.example.hn;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.*;
+import org.springframework.web.servlet.ModelAndView;
+
+import java.sql.SQLException;
+
+@Controller
+@RequestMapping("/users")
+public class UserController {
+    @Autowired
+    private UserService userService;
+
+    @GetMapping("/get")
+    public String getUserInfo(@RequestParam int id, Model model) {
+        User user = userService.getUserInfo(id);
+        if (user != null) {
+            model.addAttribute("message", true);
+            model.addAttribute("user", user);
+        } else {
+            model.addAttribute("message", false);
+            model.addAttribute("user", "The user with ID " + id + " not found");
```

Comment:
Strange key for the message.
I think, we need to put `"user"` in `if` block and `"message"` in `else` block and modify jsp accordingly